StickyTabBarViewController/Classes/ExpandableViewController.swift

```diff
@@ -21,7 +21,7 @@ public extension Expandable {
     }
 }
 
-internal class ExpandableViewController: UIViewController {
+class ExpandableViewController: UIViewController {
```

Comment:
Sometimes I use internal to make something obvious to the next reader but I don't see that's the case here. Unless I am missing something :)

Reply:
Yeah I just never used internal in my life before so wanted to be fancy 😂 @Thurman1776-

Reply:
😂 🧐